scanpy/ann_data.py

```diff
@@ -91,21 +92,20 @@ def _check_dimensions(data, smp, var):
                          .format(nr_var, var.shape[0]))
 
 class AnnData(IndexMixin):
-    def __init__(self, X, smp=None, var=None, vis=None, **meta):
-        u"""
-        Annotated Matrix
+    def __init__(self, ddata=None, X=None, smp=None, var=None, vis=None, **meta):
+        """
+        Annotated Data
 
-        Represents a sample × variable matrix (e.g. cell × gene) with the
-        possibility to store an arbitrary number of annotations for both
-        samples and variables.
+        Stores a data matrix X of dimensions nr_samples x nr_variables,
```

Comment:
yeah: `# coding: utf-8` and an `u` in front fo the docstring.

> But then, everything should be done the same way

i don’t understand what you mean by that. everything *is* done the same way. the default encoding in python 2 is ASCII, which is a subset of utf-8, which means that having or not having the UTF-8 encoding declaration is irrelevant as long as there is no immediate error.